admin/src/scenes/missions/edit.js

```diff
@@ -67,8 +90,7 @@ export default (props) => {
     initReferents();
   }, [structure]);
 
-  if ((!defaultValue && !isNew) || !structure) return <Loader />;
-
+  if ((!defaultValue && !isNew) || !structure || !structures) return <Loader />;
```

Comment:
```suggestion
  if ((!defaultValue && !isNew) || !structure) return <Loader />;
```
ne pas bloquer l'affichage de la page si on mets du temps a fetch toutes les structures